.travis.yml

```diff
@@ -2,14 +2,28 @@ language: java
 
 matrix:
   include:
-    - env: JDK='OracleJDK 8'
+    - env: JDK='Oracle JDK 8'
       jdk: oraclejdk8
-    - env: JDK='OracleJDK 9'
+    - env: JDK='Oracle JDK 9'
       jdk: oraclejdk9
-    - env: JDK='OracleJDK 10'
-      install: . ./.buildscript/install-jdk.sh -F 10 -L BCL
+    - env: JDK='Oracle JDK 10'
+      jdk: oraclejdk10
     - env: JDK='OpenJDK 10'
-      install: . ./.buildscript/install-jdk.sh -F 10 -L GPL
+      install: . ./install-jdk.sh -F 10 -L GPL
+    - env: JDK='Oracle JDK 11'
+      install: . ./install-jdk.sh -F 11 -L BCL
+    - env: JDK='OpenJDK 11'
+      install: . ./install-jdk.sh -F 11 -L GPL
+  allow_failures:
+    # Oracle JDK is not yet available on Travis CI: https://github.com/travis-ci/travis-ci/issues/9368
+    - env: JDK='Oracle JDK 10'
+    # ErrorProne/javac is not yet working on JDK 11
+    - env: JDK='Oracle JDK 11'
+    - env: JDK='OpenJDK 11'
+
+before_install:
+  - unset _JAVA_OPTIONS
+  - wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```

Comment:
Can we pin this to a SHA rather than a branch?

Reply:
You'll lose the auto-update feature/service I provide. Oracle/Java.net keep changing the URLs now and then...

The manual loading of `install-jdk.sh` might be superseded by the integration in Travis CI, soon. See https://github.com/travis-ci/travis-build/pull/1347 for details.

Reply:
Just released `1.0.1`. So we may pin it to:

`https://github.com/sormuras/bach/raw/1.0.1/install-jdk.sh`